src/lib/Preview.js

```diff
@@ -1583,14 +1583,17 @@ class Preview extends EventEmitter {
         const contentLoadTime = Timer.get(contentLoadTag) || {};
         const previewLoadTime = Timer.get(previewLoadTag) || {};
 
+        const { elapsed: contentLoadTimeDuration, end: meaningfulContentTimestamp } = contentLoadTime;
+
         this.emitLogEvent(PREVIEW_METRIC, {
             encoding,
             event_name: LOAD_METRIC.previewLoadEvent,
             value: previewLoadTime.elapsed || 0,
             [LOAD_METRIC.fileInfoTime]: infoTime.elapsed || 0,
             [LOAD_METRIC.convertTime]: convertTime.elapsed || 0,
             [LOAD_METRIC.downloadResponseTime]: downloadTime.elapsed || 0,
-            [LOAD_METRIC.contentLoadTime]: contentLoadTime.elapsed || 0,
+            [LOAD_METRIC.contentLoadTime]: contentLoadTimeDuration || 0,
+            [LOAD_METRIC.meaningfulContentTimestamp]: meaningfulContentTimestamp,
```

Comment:
Should this be `contentLoadTimestamp` to stay consistent with its other attribute? Should it default to `0`, as well?